### PR TITLE
checkbox to hide dev green data

### PR DESCRIPTION
**Pull request by sghiaa**
#### Summary of changes
**Asana Ticket:** [[2] analyzer - add checkbox for "include dev green"](https://app.asana.com/0/584764604969369/1116467262379013)

im not sure im happy with the way i did this, but i think doing it another way might be more annoying, and possibly require page reloads.  Originally i wanted to do a thing where we would just take the dev green data off the page entirely, but that would definitely require a page reload and that is nto what we wanted.

#### Reviewer Checklist
- [ ] Meets ticket's acceptance criteria
- [ ] Any new or changed functions have typespecs
- [ ] Tests were added for any new functionality (don't just rely on Codecov)

### CODE REVIEW

File: assets/js/chart.js [hunk 1]
```diff
@@ -7,6 +7,18 @@ export default function() {
       setupDashboard();
     }
 
+    jQuery('#show-dev-green-check').change(function() {
```
Comment (phildarnowsky) [inline assets/js/chart.js]: @sghiaa you can use `$` instead of `jQuery`

Comment (phildarnowsky) [reply assets/js/chart.js]: @sghiaa also it doesn't matter much in this case, but it's best to pass an `event` parameter to the callback and have it prevent the default behavior:

```
$('#show-dev-green-check').change(function(event) {
  event.preventDefault();
  
  // and more code
}

Comment (sghiaa) [reply assets/js/chart.js]: i had been trying iwth $ in the console and it wasnt working, but i guess i didnt check in this file - if i remember correctly they are different contexts, so i could believe it works here.

Comment (sghiaa) [reply assets/js/chart.js]: well i just tried it and no luck.  is there something i should do i.e. $ = jQuery; or something? or is it something that doesnt really matter and i can leave as jQuery?

**Pull request merged**